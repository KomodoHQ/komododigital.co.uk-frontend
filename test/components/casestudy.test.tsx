import React from 'react';
import CaseStudy from '../../src/components/casestudy';
import renderer from 'react-test-renderer';

test('CaseStudy element is rendered', () => {
  const component = renderer.create(<CaseStudy subtitle={"Test"} title={"Test"} link={"Test"} image={{childImageSharp:{fluid: {
    aspectRatio: 1,
    width: 1,
    height: 1,
    src: "",
    srcSet:"",
    sizes: ""
  }}}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
