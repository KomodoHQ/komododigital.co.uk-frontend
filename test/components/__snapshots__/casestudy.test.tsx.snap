// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseStudy element is rendered 1`] = `
<div
  className="komodoGridWrapper case-study-link-wrapper"
>
  <div
    className="case-study-link"
  >
    <div
      className="img"
    >
      <div
        className=" gatsby-image-wrapper"
        style={
          Object {
            "overflow": "hidden",
            "position": "relative",
          }
        }
      >
        <div
          style={
            Object {
              "paddingBottom": "100%",
              "width": "100%",
            }
          }
        />
        <picture>
          <source
            sizes=""
            srcSet=""
          />
          <img
            alt=""
            onLoad={[Function]}
            src=""
            style={
              Object {
                "height": "100%",
                "left": 0,
                "objectFit": "cover",
                "objectPosition": "center",
                "opacity": 1,
                "position": "absolute",
                "top": 0,
                "transition": "opacity 0.5s",
                "width": "100%",
              }
            }
            title="Test"
          />
        </picture>
        <noscript
          dangerouslySetInnerHTML={
            Object {
              "__html": "<picture><img width=\\"1\\" height=\\"1\\" src=\\"\\" alt=\\"\\" title=\\"Test\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
            }
          }
        />
      </div>
    </div>
    <div
      className="content"
    >
      <span>
        Test
      </span>
      <h2>
        Test
      </h2>
      <a
        href="/Test"
        onClick={[Function]}
      >
        Read Case Study
      </a>
    </div>
  </div>
</div>
`;
